### ISSUE TITLE
Advice: add mocker api config like `mocker.config.js`

### ISSUE THREAD

**Issue by vanpipy** [jaywcjlove/mocker-api#144]
Hi, I used the independent mode like `mocker mock/index.js` and found that the host and port cannot change in this mode. So I feel it's not easy to use as a part of a project. Try to get the `HOST` and `PORT` from a config file will be better than using the `process.env`.

Is this useful or need that? 

Thanks.

**Comment by jaywcjlove** [comment]
@vanpipy I don't know what you are going to do.

**Comment by vanpipy** [comment]
Like this

```diff

-   mocker mock/index.js
+   mocker --config mocker.config.json mock/index.js // or
+   mocker mock/inde.js // search the mocker.config.json in the project defaulty

```'

Is that necessary?

**Comment by jaywcjlove** [comment]
@vanpipy Support `mocker mock/m1.js test/m2.js`

**Comment by jaywcjlove** [comment]
@vanpipy Upgrade `+ mocker-api@2.4.0`

**Comment by vanpipy** [comment]
Hi, @jaywcjlove ,

I got your idea, the next PR will upgrade the version and the document together.

**Comment by jaywcjlove** [comment]
@vanpipy Already supported. 

- [x] Support command line multi-file listeners 2c4dd5c

**Comment by jaywcjlove** [comment]
@vanpipy Support setting configuration on package.json?

```js
{
  ...
  "mocker": {
    "host": "127.0.0.1",
    "port": 7788
  }
}

```

**Comment by vanpipy** [comment]
> @vanpipy Support setting configuration on package.json?
> 
> ```js
> {
>   ...
>   "mocker": {
>     "host": "127.0.0.1",
>     "port": 7788
>   }
> }
> ```

That is a better choice :D.

**Comment by jaywcjlove** [comment]
@vanpipy Support setting parameters: 

```shell
mocker mock/index.js --port 7788
mocker mock/index.js --host 0.0.0.0
```

help: 

```shell
Usage: mocker <path> [--config] [--help|h]

Displays help information.

Options:
  --config <path> Simple configuration

Example:
  mocker mock/index.js
  mocker mock/index.js --port 7788
  mocker mock/index.js --host 0.0.0.0
  mocker mock/m1.js test/m2.js
  mocker mock/m1.js --config mocker.config.json
``

**Comment by vanpipy** [comment]
Hi, @jaywcjlove ,

Yes, `--host` and `--port` is useful, and the usage needed.

**Comment by jaywcjlove** [comment]
Hi @vanpipy 

I hope to help improve the documentation when I have time, thank you!

Command line supports more [parameter configuration](https://github.com/jaywcjlove/mocker-api/blob/6e0836c6146cd094878f0c239a37e8c1ded898c1/packages/core/src/bin/mocker.ts#L11-L14):

```json
{
  ...
  "mocker": {
    "host": "127.0.0.1",
    "port": 7788,
    "pathRewrite": {
      "^/api/repos/": "/repos/"
    },
    "changeHost": true,
    ...
  }
}
```

**Comment by vanpipy** [comment]
Summary:

The mocker command line support 3 parameters:

1. `--host localhost`
2. `--port 8000`
3. `--config ./mocker.config.json`

Of course, the `-h` and `--help` is in the house already.

And the priority of the config - assuming three configurations exist already.

1. Local mocker config like `mocker.config.json`.
2. The config in the `package.json` like:
```
{
    "mocker": { ... }
}
```
3. The parameters from the command line.

First, the mocker commander will try to find the `mocker.config.json`. If exists and overwrite the default config.
Second, try to take the `mocker` from `package.json`. If exists and overwrite again.
Third, try to take `host` and `port` from the command line and overwrite the host and port.

That is all things of the mocker command line doing.

**Comment by jaywcjlove** [comment]
@vanpipy thx! Released `mocker-api@2.7.0`

**Comment by vanpipy** [comment]
The issue resolved, so close it.
